honeycomb/honeycomb.go

```diff
@@ -63,8 +63,13 @@ func (e *Exporter) Close() {
 // Don't have a Honeycomb account? Sign up at https://ui.honeycomb.io/signup
 func NewExporter(writeKey, dataset string) *Exporter {
 	// Developer note: bump this with each release
-	versionStr := "0.0.4"
+	versionStr := "0.0.5"
 	libhoney.UserAgentAddition = "Honeycomb-OpenCensus-exporter/" + versionStr
+
+	libhoney.Init(libhoney.Config{
+		WriteKey: writeKey,
+		Dataset:  dataset,
+	})
 	builder := libhoney.NewBuilder()
 	builder.WriteKey = writeKey
```

Comment:
do we still need this and the line below with this initialization?

Reply:
nope! thanks - removed.